src/evict/evict_lru.c

```diff
@@ -147,36 +148,31 @@ __evict_list_clear(WT_SESSION_IMPL *session, WT_EVICT_ENTRY *e)
 {
     if (e->ref != NULL) {
         WT_ASSERT(session, F_ISSET_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU));
-        F_CLR_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU);
+        F_CLR_ATOMIC_16(e->ref->page, WT_PAGE_EVICT_LRU | WT_PAGE_IN_URGENT_QUEUE);
     }
     e->ref = NULL;
     e->btree = WT_DEBUG_POINT;
 }
 
 /*
- * __wt_evict_list_clear_page --
- *     Make sure a page is not in the LRU eviction list. This called from the page eviction code to
- *     make sure there is no attempt to evict a child page multiple times.
+ * __evict_list_clear_page_locked --
+ *     This function searches for the page on all the eviction queues (skipping the urgent queue if
```

Comment:
Typo: *in* all the eviction queues

Reply:
Updated in [47a4c7d](https://github.com/wiredtiger/wiredtiger/pull/8950/commits/47a4c7d30572d509ef1c3c1bf5b607e044428006)